wandb/sdk/wandb_run.py

```diff
@@ -3186,7 +3233,8 @@ def link_model(
                     - `/local/directory/file.txt`
                     - `s3://bucket/path`
             registered_model_name: (str) - the name of the registered model that the model is to be linked to. The entity will be derived from the run
```

Comment:
```suggestion
            registered_model_name: (str) - the name of the registered model that the model is to be linked to. A registered model is a collection of model versions linked to the model registry, typically representing a team's specific ML Task. The entity that this registered model belongs to will be derived from the run previously set in this context. 
```

Reply:
`previously set in this context. ` is not needed.